src/main/java/com/rarchives/ripme/ripper/rippers/WordpressComicRipper.java

```diff
@@ -133,31 +202,78 @@ public Document getNextPage(Document doc) throws IOException {
+            || getHost().contains("themonsterunderthebed.net")
+            || getHost().contains("prismblush.com")
+            || getHost().contains("www.konradokonski.com")
+            || getHost().contains("thisis.delvecomic.com")) {
                 Element elem = doc.select("div.comic-table > div#comic > a > img").first();
                 // If doc is the last page in the comic then elem.attr("src") returns null
                 // because there is no link <a> to the next page
                 if (elem == null) {
-                    logger.debug("Got last page in totempole666 comic");
                     elem = doc.select("div.comic-table > div#comic > img").first();
                 }
+                // Check if this is a site where we can get the page number from the title
+                if (url.toExternalForm().contains("buttsmithy.com") == true) {
+                    // Set the page title
+                    pageTitle = doc.select("meta[property=og:title]").attr("content");
+                    pageTitle = pageTitle.replace(" ", "");
+                    pageTitle = pageTitle.replace("P", "p");
+                }
+                if (url.toExternalForm().contains("www.totempole666.com") == true) {
+                    String postDate = doc.select("span.post-date").first().text().replaceAll("/", "_");
+                    String postTitle = doc.select("h2.post-title").first().text().replaceAll("#", "");
+                    pageTitle = postDate + "_" + postTitle;
+                }
+                if (url.toExternalForm().contains("themonsterunderthebed.net") == true) {
+                    pageTitle = doc.select("title").first().text().replaceAll("#", "");
+                    pageTitle = pageTitle.replace("“", "");
+                    pageTitle = pageTitle.replace("”", "");
+                    pageTitle = pageTitle.replace("The Monster Under the Bed", "");
+                    pageTitle = pageTitle.replace("–", "");
+                    pageTitle = pageTitle.replace(",", "");
+                    pageTitle = pageTitle.replace(" ", "");
+
+                }
                 result.add(elem.attr("src"));
             }
+
+
+            // freeadultcomix gets it own if because it needs to add http://freeadultcomix.com to the start of each link
+            if (url.toExternalForm().contains("freeadultcomix.com") == true) {
+                for (Element elem : doc.select("div.single-post > p > img.aligncenter")) {
+                    result.add("http://freeadultcomix.com" + elem.attr("src"));
+                }
+            }
+
+            if (url.toExternalForm().contains("comics-xxx.com") == true) {
+                for (Element elem : doc.select("div.single-post > center > p > img")) {
+                    result.add(elem.attr("src"));
+                }
+            }
             return result;
         }
 
         @Override
         public void downloadURL(URL url, int index) {
+            sleep(500);
```

Comment:
Is there any rate limiting in play that would make a delay useful? (If not, we're just slowing down the rip for no reason.)